week_2/task1/src/utils/utils.ts

```diff
@@ -0,0 +1,45 @@
+
+import { Employee, HolidayRequest} from '../types/types';
+import { getEmployees } from './dataManager';
+
+export const employeesFilename = './data/employees.json';
+export const holidaysFilename = './data/holidays.json';
+
+
+export function increaseUserHolidays(employeeId: number, daysNum: number) {
+  const employees: Employee[] = getEmployees();
+  const user = employees.find(employee => employee.id === employeeId);
+
+  if (user) {
+    user.remainingHolidays += daysNum;
+  }
+  return employees;
+}
+
+export function getDaysNum(request: HolidayRequest) {
+  const endDate = new Date(request.endDate);
+  const startDate = new Date(request.startDate);
+
+  return (endDate.getTime() - startDate.getTime()) / (1000 * 3600 * 24) + 1
```

Comment:
Will be good make a constant for time (milliseconds in days) to make code more understandable

Reply:
Yes, we thought about that with Dmitro, but forgot to add. Thank for comment